python/cudf/benchmarks/API/bench_multiindex.py

```diff
@@ -11,16 +11,16 @@
 @pytest.fixture
 def pidx():
     num_elements = int(1e3)
-    a = np.random.randint(0, num_elements // 10, num_elements)
-    b = np.random.randint(0, num_elements // 10, num_elements)
+    a = np.random.default_rng(2).integers(0, num_elements // 10, num_elements)
+    b = np.random.default_rng(2).integers(0, num_elements // 10, num_elements)
```

Comment:
I think we want different seeds for each column. Otherwise the results should be identical when they were previously different.
```suggestion
    a = np.random.default_rng(2).integers(0, num_elements // 10, num_elements)
    b = np.random.default_rng(3).integers(0, num_elements // 10, num_elements)
```